ast/expressions.go

```diff
@@ -443,6 +444,47 @@ func (n *CompareSubqueryExpr) Accept(v Visitor) (Node, bool) {
 	return v.Leave(n)
 }
 
+// ObjectNameExpr represents a table-level name expression, such as sequence/table/view etc.
+type ObjectNameExpr struct {
```

Comment:
I think renaming to `TableNameExpr` is better.

Reply:
Fine, addressed.